src/app/header/header.component.html

```diff
@@ -1,74 +1,157 @@
-<nav class="navbar navbar-default navbar-pf" role="navigation" *ngIf="loggedIn">
-    <div class="navbar-header"></div>
+<nav class="navbar navbar-inverse navbar-pf" role="navigation" *ngIf="loggedIn">
```

Comment:
We do have a dropdown menu component already. It is available in the shared-component folder. Will that work for you?